src/app/file-manager/reducers/export-to-s3.reducer.ts

```diff
@@ -204,23 +204,23 @@ export function ExportToS3Reducer(
             break;
 
         case ExportToS3ItemActions.UPDATE_ITEM_PROGRESS:
-          if (state.inProgress.items[action.payload.id] !== undefined) {
-            // decrement old values
-            state.totalTransferred -= state.inProgress.items[action.payload.id].transferred;
-            state.inProgress.transferred -= state.inProgress.items[action.payload.id].transferred;
+            if (state.inProgress.items[action.payload.id] !== undefined) {
```

Comment:
The entire file is using the wrong indent; the code uses 2 spaces per indent, not 4.